website/meta/sidebars.json

```diff
@@ -122,7 +122,8 @@
                     { "text": "Scorer", "url": "/api/scorer" },
                     { "text": "StringStore", "url": "/api/stringstore" },
                     { "text": "Vectors", "url": "/api/vectors" },
-                    { "text": "Vocab", "url": "/api/vocab" }
+                    { "text": "Vocab", "url": "/api/vocab" },
+                    { "text": "Legacy", "url": "/api/legacy" }
                 ]
```

Comment:
This list is otherwise alphabetized so this is a bit weird. Maybe it could go at the end of "Overview" instead?

Reply:
It feels perhaps a bit too prominent in Overview? And it'd push down other more import parts :(

Reply:
We could also have an entire separate section all the way at the bottom (perhaps before CYTHON), assuming we'll be subdividing this at some point anyway...

Reply:
I think a separate section sounds good.